vnext/src/Libraries/Text/Text.windows.js

```diff
@@ -153,6 +153,7 @@ const Text: React.AbstractComponent<
         {...restProps}
         {...eventHandlersForText}
         isHighlighted={isHighlighted}
+        isPressable={isPressable}
```

Comment:
Shouldn't you also add this to <NativeText> instances in this file? Lines 203, 220

Reply:
No, we actually only need `isPressable` for `RCTVirtualText`. Pressability on the top-level Text component is already supported by the existing TouchEventHandler algorithm.